backend/src/main/java/capstone/herbs/plants/HerbsRepository.java

```diff
@@ -0,0 +1,13 @@
+package capstone.herbs.plants;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface HerbsRepository extends MongoRepository<HerbsItem, String> {
+    Optional<HerbsItem> findAllByHerbsName(String herbsName);
+
+
+}
```

Comment:
```suggestion
public interface HerbsRepository extends MongoRepository<HerbsItem, String> {
    Collection<HerbsItem> findAllByHerbsName(String herbsName);
}
```
As Bennet said, but you don't use it, so you could also delete it